packages/main/src/Link.js

```diff
@@ -102,6 +102,19 @@ const metadata = {
 			defaultValue: WrappingType.None,
 		},
 
+		/**
+		 * Sets the accessible aria name of the component.
+		 *
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @public
+		 * @since 1.2.0
+		 */
+		accessibleName: {
+			type: String,
+			defaultValue: "",
```

Comment:
This is not needed. By default it will be undefined

Reply:
It should be `empty string` by default, rather than `undefined`, but anyway same - you can omit `defaultValue: ""` (it's sufficient to have it in the JS Docs, so that it is displayed in the API reference).

```js
// UI5ElementMetadata.js

if (propType === String) {
  initialState[propName] = "defaultValue" in props[propName] ? props[propName].defaultValue : "";
}
```